.github/workflows/lint-and-test.yml

```diff
@@ -77,29 +84,43 @@ jobs:
     - name: Install PHP dependencies
       run: |
         export PATH="$HOME/.composer/vendor/bin:$PATH"
-        composer install --no-interaction
+        composer install --no-dev --no-interaction
         ulimit -n 4096
 
     - name: Run ESLint and Stylelint
       run: npm run lint
       if: matrix.experimental == false
 
     - name: Run PHP CodeSniffer
-      run: composer standards
+      run: |
+        composer update --dev pressbooks/coding-standards
+        composer standards
+      if: matrix.experimental == false
 
     - name: Install WP tests
       run: bash bin/install-wp-tests.sh wordpress_test root root localhost ${{ matrix.wordpress }}
 
+    - name: Install PHPUnit
+      run: composer update --dev yoast/phpunit-polyfills --with-dependencies --ignore-platform-reqs
```

Comment:
Installs relevant version of PHPUnit (will depending on WP version being tested)